lib/semantics/resolve-names.cc

```diff
@@ -2790,6 +2791,22 @@ Symbol &SubprogramVisitor::PushSubprogramScope(
   return *symbol;
 }
 
```

Comment:
It would be nice if we could know what clause of the standard are you implementing

Reply:
`BLOCK DATA` is in clause 14.3.